src/PointwiseFunctions/AnalyticData/GrMhd/BlastWave.cpp

```diff
@@ -18,32 +20,34 @@
 
 namespace {
 template <typename DataType>
-Scalar<DataType> compute_piecewise(const tnsr::I<DataType, 3>& x,
-                                   const double inner_radius,
-                                   const double outer_radius,
-                                   const double inner_value,
-                                   const double outer_value) noexcept {
-  const DataType cylindrical_radius =
-      sqrt(square(get<0>(x)) + square(get<1>(x)));
+Scalar<DataType> compute_piecewise(
+    const tnsr::I<DataType, 3>& x,
+    const grmhd::AnalyticData::BlastWave::Geometry geometry,
```

Comment:
[super nitpick, feel free to veto] Put the Geometry argument last to match all other functions